### DISCUSSION TITLE
Bug report Thefreedictionary.com

### DISCUSSION THREAD

**Discussion by makamoe12** [aclap-dev/video-downloadhelper#689] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://de.thefreedictionary.com/klarstellen",
  "error": {
    "details": "SideDownload error: : [tcp @ 0000024c865ab280] Connection to tcp://img2.tfd.com:443 failed: Error number -138 occurred\r\n[in#0 @ 0000024c87bdb2c0] Error opening input: Error number -138 occurred\r\nError opening input file https://img2.tfd.com/pron/mp3/de/DE/d3/d3d5d3dgsdsysod5sgsfdnsth7h3.mp3.\r\nError opening input files: Error number -138 occurred\r\n",
    "downloadable_id": "downloadable_5283958373953038",
    "error": "coapp_failure",
    "id": "coapp_failure_c19cb32e-f69d-47c7-887e-3a916419f8f8"
  }
}

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: es-ES
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\Usuario\\Desktop"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "default_action": "download_as",
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 5,
    "successfull_dl": 15
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

I clicked the little speaker icon on that page.  I'm assuming that's supposed to give you a short clip of somebody pronouncing the word.  After clicking it a couple of times & hearing nothing, I noticed VDH had finally put something in its menu that had initially been empty.  I downloaded it.  It was just a couple of seconds long & it seemed to be playing in VLC but I still didn't hear anything.  Cranking up the volume finally made it audible.  I imagine that's all I had to do to make it audible in the web page as well.

So I say there's nothing wrong here.  It's working fine.  This may be yet another example of Firefox doing a better job with something than Chrome.

**Reply by makamoe12** [reply]
Thanks for your reply. Your are correct: clicking on the speaker icon plays an .mp3 file with the pronounciation of the word. That is the file I intended to download. However, I can hear the audio fine when I click on it, with a normal volume. This is the file it's supposed to play/download: https://img2.tfd.com/pron/mp3/de/DE/d3/d3d5d3dgsdsysod5sgsfdnsth7h3.mp3

VDH had worked fine until yesterday, but now gives the bug report I pasted when I try to download the file.
